src/utils/util/resolve/punchlist.md

```diff
@@ -0,0 +1,16 @@
+# Profile resolver punchlist
+
+Things to sketch out and test:
+
+Import with no include - implicit all
```

Comment:
I think you already have this as an example?

Reply:
This is now in the experimental pipeline with clearer docs.